litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/TableToolbar.tsx

```diff
@@ -155,10 +155,10 @@ const TableToolBar: React.FC<TableToolBarProps> = ({
           <Typography variant="h6" component="div" display="inline">
             <strong>
               &nbsp; {numSelected}{' '}
-              {numSelected === 1 ? 'workflow' : 'workflows'} selected
+              {numSelected === 1 ? 'workflow' : 'workflows'} selected &nbsp;
```

Comment:
same

Reply:
This is also a part of the table toolbar if we change this we may want to change it for all tables even all table data should be translated. having header in the native language and table data in English won't be a good UX. @oumkale

Reply:
For all the tables currently, data and headers are in English, we might consider making this change for all table at once after the decision. @oumkale